src/osgPlugins/OpenFlight/AncillaryRecords.cpp

```diff
@@ -276,23 +355,20 @@ class UVList : public Record
             uint32 mask = in.readUInt32(0);
 
             int numLayers = bitCount(mask);
-            if (numLayers>0)
```

Comment:
I think we want these changes. It is just checking to make sure the number of layers is greater than 0.